Examples/Tests/embedded_circle/inputs_2d

```diff
@@ -0,0 +1,80 @@
+max_step = 1
```

Comment:
Since this is a rather complex test and its name is not self-explanatory it would be probably a good idea to explain in a few lines what the test does.

Reply:
Sounds good, I'll add that in.